coordinator/src/test/java/io/narayana/lra/coordinator/domain/model/LRATestBase.java

```diff
@@ -187,7 +187,6 @@ public Response actionWithLRA(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI contextI
         public Response timeLimitTest(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId) {
             try {
                 Thread.sleep(1000); // sleep for longer than specified in the timeLimit annotation attribute
-                // TODO use byteman
```

Comment:
This commit is about log messages but here you appear to have decided not to use byteman anymore? Please can you let me know it is intentional?

Reply:
I think it is clear which tests need byteman support (for predictable testing) so I removed it.

Reply:
That's fine with me, then. This specific change probably should be outside of the commit worded "JBTM-3945 clean up warning messages" but if it doesn't move outside of the commit I am OK with that. Thank you for explaining.